code-challenge/401/array_shift/array-shift.js

```diff
@@ -0,0 +1,22 @@
+function insertShiftArray(item) {
```

Comment:
problem domain says a function that takes an array and a value to be added. Your function takes only one argument.

Reply:
Ed, re-submit your assignment with unit tests as well